Engineering/Management/Pomodoro Technique.md

```diff
@@ -0,0 +1,29 @@
+  
+
+---
+
+tags: engineering, management, time
+
+author: Tran Hoang Nam
+
+---
+
```

Comment:
Avoid adding too many spaces here, otherwise both Markdown and Obsidian will not recognize this as metadata or frontmatter.